packages/twenty-front/src/modules/object-record/record-board/components/RecordBoard.tsx

```diff
@@ -61,6 +62,7 @@ export const RecordBoard = () => {
     columnIdsState,
     columnsFamilySelector,
     recordIdsByColumnIdFamilyState,
+    allRecordIdsSelector,
```

Comment:
we could refactor this into record-index to mutualize this behavior between table and board but I don't think this is the topic of this PR